Binary files orig/BHC/src/.Makevars.swp and patched/BHC/src/.Makevars.swp differ
diff -Nru orig/BHC/src/Makevars patched/BHC/src/Makevars
--- orig/BHC/src/Makevars	2024-01-11 22:14:33.000000000 +0100
+++ patched/BHC/src/Makevars	2024-03-16 11:42:44.481858700 +0100
@@ -1,2 +1,3 @@
+CXX_STD = CXX14
 PKG_CPPFLAGS=-fopenmp
 PKG_LIBS=-fopenmp 
diff -Nru orig/BHC/src/Makevars.in patched/BHC/src/Makevars.in
--- orig/BHC/src/Makevars.in	2023-10-20 20:16:08.000000000 +0200
+++ patched/BHC/src/Makevars.in	2024-03-16 11:42:39.450980600 +0100
@@ -1,2 +1,3 @@
+CXX_STD = CXX14
 PKG_CPPFLAGS=@OPENMP_CFLAGS@
 PKG_LIBS=@OPENMP_CFLAGS@ @LIBS@
